local-cli/cli.js

```diff
@@ -24,7 +24,8 @@ function printUsage() {
     '  install: installs npm react components',
     '  bundle: builds the javascript bundle for offline use',
     '  new-library: generates a native library bridge',
-    '  android: generates an Android project for your app'
+    '  android: generates an Android project for your app',
+    '  run-android: open app in the Android simulator'
```

Comment:
This is not necessarily a simulator, it can be a device. Also on Android these are called emulators.